#! /bin/sh /usr/share/dpatch/dpatch-run
## 03.latex8_counters.dpatch by Joe Pfeiffer <pfeiffer@cs.nmsu.edu>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add counters to the layout8 article class.

@DPATCH@
diff -urNad lyx-1.3.6/lib/layouts/latex8.layout /tmp/dpep.Zz5xpO/lyx-1.3.6/lib/layouts/latex8.layout
--- lyx-1.3.6/lib/layouts/latex8.layout	1999-09-28 04:44:33.000000000 +1000
+++ /tmp/dpep.Zz5xpO/lyx-1.3.6/lib/layouts/latex8.layout	2005-09-13 16:15:38.769178408 +1000
@@ -123,3 +123,13 @@
     Size		Large
   EndFont
 End
+
+# nonstandard counters
+
+Counter
+       Name    Section
+End
+
+Counter
+       Name    SubSection
+End


